src/internal/clusterstate/v2.11.0/pjs.go

```diff
@@ -0,0 +1,55 @@
+package v2_11_0
+
+import (
+	"context"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+)
+
+func createPJSSchema(ctx context.Context, env migrations.Env) error {
+	ctx = pctx.Child(ctx, "createPJSSchema")
+	tx := env.Tx
+	_, err := tx.ExecContext(ctx, `
+		CREATE SCHEMA pjs; 
+		CREATE TYPE pjs.job_error_code AS ENUM (
+			'failed', 
+			'disconnected',
+			'canceled'
+		);
+		CREATE TABLE pjs.jobs (
+				id BIGSERIAL PRIMARY KEY,
+				parent BIGINT REFERENCES pjs.jobs(id),
+				
+				program BYTEA NOT NULL,
+				program_hash BYTEA NOT NULL, -- the hash will be used to model the queues
+				
+				error pjs.job_error_code,
+				
+				queued timestamptz DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
I think we have a pattern of doing `x_at` `y_at`, so `queued_at` `run_at` `done_at`.

Reply:
We talked about dropping the *_at suffix at the design review. I don't have a strong opinion on it, but that's where the naming comes from.